From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@danielsollondon__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

andredm7 committed:
Changes to RBAC

ktoliver committed:
edit pass: storage-files-faq.md

ktoliver committed:
Update storage-files-faq.md

wmgries committed:
Revert FAQ formatting changes.

wmgries committed:
Revert base-10 units (MB, GB, TB) to base-2 units (MiB, GiB, TiB), which is what Azure Files uses.

wmgries committed:
Revert "Azure Files share" to "Azure File share". Agreed to naming convention.

wmgries committed:
Blob storage is capitalized (see linked article).

wmgries committed:
Revert removal of the acronyms. These are used extensively throughtout our docs and blog posts, so customers need to understand them.

wmgries committed:
Remove "blob" from "cool storage". Azure Files is not Blob storage.

wmgries committed:
Remove stray quote.

wmgries committed:
Re-add storage redundancy acronyms.

wmgries committed:
Data Deduplication is a Windows Server feature, and is capitalized.

wmgries committed:
Add SAS acronym - extensively referenced in documentation, customers need to learn it.

wmgries committed:
Fix existing typo in AFS ACL sync support statement.

ktoliver committed:
Merge pull request #3 from wmgries/storage-files-faq-B

Quick fixes to Azure Files FAQ review

ktoliver committed:
Update storage-files-faq.md

more edits

ktoliver committed:
Update storage-files-faq.md

lowercase sync group

ktoliver committed:
Update storage-files-faq.md

Azure File share > Azure file share

curtand committed:
work info merged with user profile

curtand committed:
work info merged with user profile

veljko-msft committed:
Update sql-database-automatic-tuning-enable.md

Adding other options for configuring automatic tuning

Commit:
LP update due to TOC update

SteveLas committed:
Admin and Role Based Access updates

veljko-msft committed:
Update sql-database-automatic-tuning-enable.md

Commit:
retirement

Commit:
retirement

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into classic1

mmacy committed:
[aci] restore PowerShell ref links

spelluru committed:
updated delete api

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into flores1102

curtand committed:
fix TOC link

MicrosoftGuyJFlo committed:
SaaSAppAtlassianCloud

carolinacmoravia committed:
c6866: Hardcoded EN locale in link 

Hello, @abgreg,
Reported link is hardcoded so it can lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

rwike77 committed:
quickstart updates

Commit:
Merge pull request #27094 from carolinacmoravia/patch-425

C6866: Hardcoded EN locale in link

ShawnJackson committed:
Merge pull request #26529 from ktoliver/storage-files-faq-B

edit pass: storage-files-faq.md

andredm7 committed:
Last change

v-shils committed:
Merge pull request #26866 from veljko-msft/patch-6

Update sql-database-automatic-tuning-enable.md

rwike77 committed:
more updates

Commit:
Merge pull request #27085 from spelluru/adfpython1107

updated delete api

Commit:
Added section about firewall.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasfirewall

rwike77 committed:
more updates

Commit:
add two new troubleshooting bash scenarios

PRMerger20 committed:
Merge pull request #27105 from Minewiskan/aasfirewall

Aasfirewall

Commit:
Merge pull request #27108 from jluk/jluk-cloud-shell

add two new troubleshooting bash scenarios

v-shils committed:
Merge pull request #26950 from v-savila/upd1116619

LP update due to TOC update

sethmanheim committed:
Freshness update

nberdy committed:
Update tutorial-set-up-device.md

Updating broken link

GitHubber17 committed:
Merge pull request #26791 from curtand/flores1102

work info merged with user profile

v-shils committed:
Merge pull request #27093 from MicrosoftGuyJFlo/SaaSAppAtlassianCloud

SaaSAppAtlassianCloud

JasonWHowell committed:
Update retrain-existing-resource-manager-based-web-service.md

PRMerger10 committed:
Merge pull request #27097 from rwike77/usability

quickstart updates

ktoliver committed:
Merge pull request #27076 from mmacy/aci-powershell-link-revert

[aci] restore PowerShell ref links

sethmanheim committed:
Freshness updates

Commit:
Merge pull request #27116 from nberdy/patch-1

Update tutorial-set-up-device.md

sethmanheim committed:
Acrolinx

PRMerger20 committed:
Merge pull request #27115 from sethmanheim/eh-fresh

Freshness update

JasonWHowell committed:
Update retrain-existing-resource-manager-based-web-service.md

ktoliver committed:
Merge pull request #26515 from andredm7/andredm1030171

Changes to RBAC

BryanLa committed:
Freshness scan - Fix tenant definition

cychua committed:
Add clarification to localdb

PRMerger15 committed:
Merge pull request #27121 from sethmanheim/sb-fresh

Freshness updates

cychua committed:
Add grammar improvement

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger17 committed:
Merge pull request #27123 from BryanLa/patch-14

Freshness scan - Fix tenant definition

PRMerger13 committed:
Merge pull request #27119 from JasonWHowell/patch-2

Update retrain-existing-resource-manager-based-web-service.md

v-alje committed:
Merge pull request #27127 from MicrosoftDocs/FromPublicRepo

11/7 PM Publish

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into classic1

PRMerger15 committed:
Merge pull request #26954 from SteveLas/stevelas-edits

Admin and Role Based Access updates

PRMerger18 committed:
Merge pull request #27126 from cychua/master

Add clarification to localdb

PRMerger16 committed:
Merge pull request #27053 from cynthn/classic1

Classic portal retirement

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
